burnman/minerals/other.py

```diff
@@ -12,10 +12,34 @@
 
 from .. import mineral_helpers as helpers
 from ..mineral import Mineral
+from ..solidsolution import SolidSolution
+from ..solutionmodel import *
 
 from .SLB_2011 import periclase, wuestite, mg_perovskite, fe_perovskite
 
-
+class make_endmember(SolidSolution):
```

Comment:
> I don't want the user to be able to call set_composition.

But your class is derived from SolidSolution, so these function can be called! You are creating a class derived from SolidSolution so it inherits all functionality. Of course you can add/overwrite certain functionality but you don't do that right now.